rpp.c

```diff
@@ -197,6 +199,11 @@ parse_configuration_line(struct rpp *env, const char *key, char *val)
         if (errstr != NULL) {
             errx(1, "invalid ping ttl: %s", val);
         }
+    } else if (strcasecmp(key, "retries") == 0) {
+        env->retries = (int)strtonum(val, 1, 255, &errstr);
+        if (errstr != NULL) {
+            errx(1, "invalid retry value: %s", val);
```

Comment:
would change it to "invalid retries value", to use the same 'word' as the provided key.